src/Toolkit/Html.php

```diff
@@ -175,6 +175,8 @@ public static function email(string $email, $text = null, array $attr = []): str
      * @param string|null $string
      * @param bool $keepTags If true, existing tags won't be escaped
      * @return string The HTML string
+     *
+     * @psalm-suppress ParamNameMismatch
```

Comment:
Should we add a `@todo` tag here?

Reply:
I explicitly left it out here as the param name mismatch is a non-issue for public static methods in static-only classes like `Html` and `Xml`. You wouldn't depend on `Html` being 100 % compatible with `Xml` as the classes have different use-cases. The only reason the classes use inheritance is that it reduced code duplication.